modules/azure/src/main/java/org/testcontainers/containers/AzuriteContainer.java

```diff
@@ -0,0 +1,529 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.utility.DockerImageName;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Testcontainers implementation for Azurite Emulator.
+ * <p>
+ * Supported image: {@code mcr.microsoft.com/azure-storage/azurite}
+ * <p>
+ * Exposed ports:
+ * <ul>
+ *     <li>10000 (default blob port, configurable)</li>
+ *     <li>10001 (default queue port, configurable)</li>
+ *     <li>10002 (default table port, configurable)</li>
+ * </ul>
+ * <p>
+ * See command line options <a href="https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio%2Cblob-storage#command-line-options">here</a>.
+ */
+public class AzuriteContainer extends GenericContainer<AzuriteContainer> {
```

Comment:
IMO, the container has a fast startup. By default, the three services must be started. Do you know about a specific use case to just start one of them?

As I said in the discussion, this implementation must be straightforward. For now, only expose ports would be enough.

Reply:
No, I have only assumed, that there must be a good reason behind it since the emulator's author implemented it specifically.

Reply:
Changed it to always start all 3, thank you!